package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@nthparty/opaque",
-  "version": "0.3.0",
+  "version": "0.4.0-beta.1.0",
   "description": "TypeScript implementation of the OPAQUE asymmetric password authenticated key exchange (aPAKE) protocol",
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc -p tsconfig.prod.json && cp -r src/types dist/types",
     "test": "jest --coverage",
-    "coveralls": "npm test && ./node_modules/coveralls/bin/coveralls.js < coverage/lcov.info"
+    "lint": "eslint src"
```

Comment:
I might have missed moving the `coveralls` script elsewhere, but it seems to me that it was deleted. If so, then it may be best to uninstall the `coveralls` package as well.

That said, .travis.yml references this script, so maybe leave it in?